packages/@react-spectrum/provider/docs/Provider.mdx

```diff
@@ -0,0 +1,160 @@
+  sourceData={[]} />
+
+## Example
+
+```tsx example
+<Provider isDisabled scale="large">
+  <Button variant="primary">Button</Button>
+</Provider>
+```
+
+## Children Props
+
+As a convenience, Provider supports passing common props to all components
+within it. For example, an entire group of components could be disabled by
+setting `isDisabled` in one place rather than on each individual element.
+These props are `isDisabled`, `isEmphasized`, `isQuiet`, `isReadOnly`,
+`isRequired`, and `validationState`.
+
+### Quiet
+
+```tsx example
+<Provider isQuiet>
+  <Form>
+    <div>
+      <TextField label="Email" />
+    </div>
+    <Checkbox>
+      Checkbox Label
+    </Checkbox>
+    <div>
+      <Button variant="primary">Button</Button>
+    </div>
+  </Form>
+</Provider>
+```
+
+## Global Settings
+
+Props such as `theme`, `colorScheme`, `locale`, and `scale` adjust all
+components with Provider. The `theme` allows the definition of custom CSS
+variables for the `colorScheme` and `scale` prop values.
+
+### Color Scheme and Scale
+
+```tsx example
+<Provider colorScheme="dark" scale="large" UNSAFE_style={{padding: '15px'}}>
+  <Form>
+    <div>
+      <TextField label="Email" />
+    </div>
+    <Checkbox>
+      Checkbox Label
+    </Checkbox>
+    <div>
+      <Button variant="primary">Button</Button>
+    </div>
+  </Form>
+</Provider>
+```
+
+### Theme
+
+In this example the `theme` is being changed so that the "light" `theme`'s css
+is being applied to the "dark" `colorScheme`.
+
+```tsx example
+function Example() {
+  /*
+   * These are imported elsewhere in the file and commented out in this example
+   * as a reminder of where the variables come from.
+   */
+  // import scaleMedium from '@adobe/spectrum-css-temp/vars/spectrum-medium-unique.css';
+  // import themeLight from '@adobe/spectrum-css-temp/vars/spectrum-light-unique.css';
+
+  let theme = {
+    dark: themeLight,
+    medium: scaleMedium
+  };
+
+  return (
+    <Provider theme={theme} colorScheme="dark">
+      <Form>
+        <div>
+          <TextField label="Email" />
+        </div>
+        <Checkbox>
+          Checkbox Label
+        </Checkbox>
+        <div>
+          <Button variant="primary">Button</Button>
+        </div>
+      </Form>
+    </Provider>
+  );
+}
+
+```
+
+### Locale
+
+The Provider can override the `locale` prop which defaults to the system/browser
+settings and changes how all children components are internationalized.
+
+```tsx example
+<Provider locale="ar-AE">
+  <Form necessityIndicator="label">
+    <div>
+      <TextField label="Email" />
+    </div>
+    <Checkbox>
+      Checkbox Label
+    </Checkbox>
+    <div>
+      <Button variant="primary">Button</Button>
+    </div>
+  </Form>
+</Provider>
+```
+
+## Provider Nesting
+
+Providers may be nested. The parent provider passes its global settings props (`theme`) down to the
+children providers, but not its context props (`isDisabled`, `isEmphasized`, etc.).
```

Comment:
Maybe change `theme` -> `scale` and remove `isEmphasized` so that the wording reflects the example below